src/containers/WelcomePage/components/LastUsedContent.tsx

```diff
@@ -0,0 +1,80 @@
+import React, { FC, useEffect, useState } from 'react';
+import { Link } from 'react-router-dom';
+import { injectT } from '@ndla/i18n';
+
+import {
+  ArticleType,
+  ContentResultType,
+  TranslateType,
+} from '../../../interfaces';
+import { RESOURCE_TYPE_LEARNING_PATH } from '../../../constants';
+import {
+  getContentTypeFromResourceTypes,
+  resourceToLinkProps,
+} from '../../../util/resourceHelpers';
+import { isLearningpath } from '../../../util/routeHelpers';
+import formatDate from '../../../util/formatDate';
+import { fetchDraft } from '../../../modules/draft/draftApi';
+import { classes } from '../WelcomePage';
+
+interface Props {
+  articleId: number;
+  content: ContentResultType;
+  locale: string;
+  t: TranslateType;
+}
+
+const LastUsedContent: FC<Props> = ({ articleId, content, locale, t }) => {
+  const [article, setArticle] = useState<ArticleType>();
+
+  const fetchArticle = async (articleId: number, locale: string) => {
+    const article = await fetchDraft(articleId, locale);
+    setArticle(article);
+  };
+
+  useEffect(() => {
+    if (articleId) {
+      fetchArticle(articleId, locale);
+    }
+  }, []);
+
+  const getResourceType = () => {
+    const resourceTypes =
+      content.contexts?.find(context => context.resourceTypes)?.resourceTypes ||
```

Comment:
skulle vi ikke sjekke `context.resourceTypes?.length` her? altså for å faktisk se om den arrayen som blir returnert har noe elementer ?

Reply:
Ellers ser det bra ut. Bra jobba!

Reply:
Usikker. Dette virket da jeg testet det og returnerte resourceTypes om de fantes, så lot det stå. 

Den "finner" jo bare context om context har en resourceType, så blir vel i teorien det samme om man sjekker med å si at om den har resourcetype og den er lenger enn 0 så "find" denne.

Reply:
Leste gjennom det igjen og ser at med mindre noen har contexts { ..., resourceType: []} , kan ta høyde for det om dere tror det er nødvendig

Reply:
ja det var det jeg mente, for tom array er true i javascript, så kanskje greit å sjekke lengden og faktisk returnere noe som er der å ikke bare tom array, for det gjør vi etter ||

Reply:
faktisk så vil koden etter || aldri bli utført. siden tom array evalueres til true, så || blir ikke utført, altså tom arrayen kommer til alltid til å komme fra .find() å ikke fra `|| []` så du må faktisk sjekke lengden i find() for å få riktig funksjonalitet

Reply:
Jo, for om ikke "find" fant noe, for context kan være tom, f.eks i de test artiklene jeg har laget. Da ble resourceType = undefined.

Reply:
det er derfor vi har ?. ;)

Reply:
Den blir undefined om ingen av de tingene med ? bak finnes, derav. Om du ikke finner noe der, sett verdien til [], slik at man kan sjekke lengden av resourceTypes ;)

Reply:
Om du lager en artikkel med bare tittel, og fjerner `|| []`  vil du få feilmeldingen `Object is possibly 'undefined'.`

Reply:
jeg ba deg ikke om å fjerne `|| []` :P

Reply:
og ja veldig bra tenkt, det skal både være resourceTypes?.length der og || [] ^^

Reply:
Du skrev jo at "faktisk så vil koden etter || aldri bli utført. " etterfulgt av "det er derfor vi har ?."  Tolket det som at det som om du mente den ikke burde være der. 

Nå er begge deler der, så nå burde det funke i forhåpentligvis meeest mulig tilfeller :)